package.json

```diff
@@ -127,6 +127,7 @@
     "prettier": "^1.14.2",
     "react-test-renderer": "^16.4.2",
     "style-loader": "^0.21.0",
+    "styled-jsx": "^3.0.2",
```

Comment:
Dunno how this managed to sneak down to `devDependencies`, move it back up to `dependencies`.

Reply:
Which ones specifically are you referring to?

Reply:
`styled-jsx`, sorry about that. We're using it during runtime, so it needs to be considered a dependency.